cli/smartcontract/smart_contract.go

```diff
@@ -205,10 +205,10 @@ func NewCommands() []cli.Command {
    Arguments always do have regular Neo smart contract parameter types, either
    specified explicitly or being inferred from the value. To specify the type
    manually use "type:value" syntax where the type is one of the following:
-   'signature', 'bool', 'int', 'hash160', 'hash256', 'bytes', 'key' or 'string'.
-   Array types are also supported: use special space-separated '[' and ']' 
-   symbols around array values to denote array bounds. Nested arrays are also 
-   supported.
+   'signature', 'bool', 'int', 'hash160', 'hash256', 'bytes', 'filebytes', key' 
```

Comment:
Maybe we better write about it in a separate sentence, it's not really a proper type after all.